### PR TITLE
Misc ui 2

### PR DESCRIPTION
**Pull request by charlesh88**
Latest TCR merged in yesterday and minor conflicts resolved.

### Changes
- Fixes plot legend layout and coloring;
- Style fixes for plot options and props in Inspector;
- Better organization for table styles;
- Fixes z-indexing in overlays;
- Fixes local controls in imagery and plots;

### Author Checklist
1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? Y
1. Changes have been smoke-tested? Y

### CODE REVIEW

File: src/ui/layout/mct-tree.vue [hunk 1]
```diff
@@ -71,7 +71,7 @@
             }
 
             &__view-control {
-                color: $colorItemTreeVC;
+                // color: $colorItemTreeVC;
```
Comment (akhenry) [inline src/ui/layout/mct-tree.vue]: Commented code

File: src/styles-new/_legacy-plots.scss [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+            height: 100%;
+        }
+        &.hash-h {
+            border-bottom: 1px $colorPlotHash $stylePlotHash;
+            width: 100%;
+        }
+    }
+
+    /****************************** Limits and Out-of-Bounds data */
+    /*
```
Comment (akhenry) [inline src/styles-new/_legacy-plots.scss]: Commented code

File: src/styles-new/_legacy-plots.scss [hunk 3]
```diff
@@ -0,0 +1,541 @@
+
```
Comment (akhenry) [inline src/styles-new/_legacy-plots.scss]: Missing copyright header

File: src/styles-new/_legacy-plots.scss [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+}
+
+.gl-plot {
+    color: $colorPlotFg;
+    display: flex;
+    font-size: 0.7rem;
+    position: relative;
+    width: 100%;
+    height: 100%;
+    //min-height: $plotMinH;
```
Comment (akhenry) [inline src/styles-new/_legacy-plots.scss]: Commented code

File: src/styles-new/_legacy-plots.scss [hunk 5]
```diff
@@ -0,0 +1,541 @@
...
+            td {
+                color: $fg;
+            }
+        }
+    }
+}
+
+/*************************************************************************** _LEGEND.SCSS */
+.gl-plot-legend {
+    //min-height: $plotLegendH;
```
Comment (akhenry) [inline src/styles-new/_legacy-plots.scss]: Commented code.

File: src/styles-new/_legacy-plots.scss [hunk 6]
```diff
@@ -0,0 +1,541 @@
...
+    &.plot-legend-expanded {
+        .gl-plot-legend {
+            max-height: 70%;
+        }
+
+        .plot-wrapper-expanded-legend {
+            overflow-y: auto;
+        }
+
+     /*   &.plot-legend-top .gl-plot-legend {
```
Comment (akhenry) [inline src/styles-new/_legacy-plots.scss]: Commented code

File: src/styles-new/_table.scss [hunk 7]
```diff
@@ -21,53 +21,53 @@
...
         vertical-align: top;
     }
+}
 
-    &__control-bar {
-        margin-bottom: $interiorMarginSm;
-    }
+.c-table {
+    // Can be used by any type of table, scrolling, LAD, etc.
+    //$min-w: 50px;
```
Comment (akhenry) [inline src/styles-new/_table.scss]: Commented code

File: src/ui/layout/mct-tree.vue (hunk 1 above)
Comment (charlesh88) [reply src/ui/layout/mct-tree.vue]: Removed.

File: src/styles-new/_legacy-plots.scss (hunk 2 above)
Comment (charlesh88) [reply src/styles-new/_legacy-plots.scss]: Cleaned up.

File: src/styles-new/_legacy-plots.scss (hunk 3 above)
Comment (charlesh88) [reply src/styles-new/_legacy-plots.scss]: Added.

File: src/styles-new/_legacy-plots.scss (hunk 4 above)
Comment (charlesh88) [reply src/styles-new/_legacy-plots.scss]: Removed comment.

File: src/styles-new/_legacy-plots.scss (hunk 5 above)
Comment (charlesh88) [reply src/styles-new/_legacy-plots.scss]: Removed.

File: src/styles-new/_legacy-plots.scss (hunk 6 above)
Comment (charlesh88) [reply src/styles-new/_legacy-plots.scss]: Removed.

File: src/styles-new/_table.scss (hunk 7 above)
Comment (charlesh88) [reply src/styles-new/_table.scss]: Removed.

File: src/styles-new/_controls.scss [hunk 8]
```diff
@@ -545,3 +601,27 @@ input[type=number]::-webkit-outer-spin-button {
...
+        right: $interiorMargin; top: $interiorMargin;
+        z-index: 2;
+    }
+
+    &-trans {
+        // Has a translucent background plate
+        background: rgba($colorBodyBg, 0.8);
+        border-radius: $controlCr;
+    }
+}
```
[the remark this answers is not in the document]
Comment (charlesh88) [reply src/styles-new/_controls.scss]: Ack, sorry. Just added line return at file end.

[the remark this answers is not in the document]
Comment (akhenry) [reply src/styles-new/_controls.scss]: You have satiated my pedantry...for now.

**Pull request merged**